### PR TITLE
[EN/ID] Update `Guides/Using_custom_hitsounds`

### PR DESCRIPTION
**Pull request by ZeroPyrozen**
## Summary
- ~Fix typo in `en.md`~ already fixed on #6485 
- Update Indonesian translation
- SKIP_OUTDATED_CHECK on other article

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Guides/Using_custom_hitsounds/id.md [hunk 1]
```diff
@@ -1,48 +1,68 @@
...
+Berkas hitsound sebaiknya berupa `.wav` atau `.ogg` yang tidak memiliki *playback delay* dan suara dapat di-*loop* secara benar. Berkas suara yang berupa `.mp3` memiliki sedikit *delay* dan mungkin tidak dapat diputar secara sinkron terhadap lagu yang ingin kamu tambahkan hitsound. Namun hitsound yang berformat `.mp3` dapat digunakan untuk efek suara khusus seperti aplaus atau suara *ambient* di mana ukuran berkas `.wav` atau `.ogg` dapat melebihi ukuran yang diregulasi.
 
-- Anda bisa mengambil dari [Custom Hitsound Library](/wiki/Guides/Custom_Hitsound_Library),
-- atau membuat kustom hitsound sendiri
-  - jika Anda membuat sendiri, pastikan format file tersebut adalah `.wav` - karena format inilah yang paling baik untuk berkas audio yang bersifat looping.
+## Menambahkan hitsound khusus
 
-Jika Anda sudah mendapatkannya, pindahkan semuanya ke folder beatmap itu sendiri. Jika Anda tidak tahu di mana letak folder beatmap tersebut, ikuti instruksi ini!:
+Setelah kamu mendapatkan berkas suara yang kamu inginkan, pindahkan berkas tersebut ke dalam folder beatmap yang kamu ingin map. Jika kamu tidak tahu di mana folder tersebut berada kamu dapat mengikuti instruksi berikut ini:
```
Comment (daud-ahn) [inline wiki/Guides/Using_custom_hitsounds/id.md:13]:
```suggestion
Setelah kamu mendapatkan berkas suara yang kamu inginkan, pindahkan berkas tersebut ke dalam folder beatmap yang kamu ingin beri hitsound. Jika kamu tidak tahu di mana folder tersebut berada kamu dapat mengikuti instruksi berikut ini:
```

File: wiki/Guides/Using_custom_hitsounds/id.md [hunk 2]
```diff
@@ -1,48 +1,68 @@
...
+`<sampleset>-<sound>.wav`
 
-Jika Anda menginginkan variasi yang lebih dan membutuhkan banyak pilihan untuk clap dan sebagainya, Anda dapat menambahkan nomor di belakangnya.
+dimana `<sampleset>` adalah "normal", "soft" atau "drum", dan `<sound>` adalah *addition* yang telah disebutkan di atas (contoh: "hitclap"). 
 
-### Contoh 2
+![](img/beatmap-folder-resources.png "Contoh folder beatmap yang berisi hitsound khusus")
 
-Seperti `normal-hitclap2.wav` atau `soft-hitfinish3.wav`
+Pada gambar yang ditunjukkan di atas, berkas pertama bernama `soft-hitclap.wav` dan akan menggantikan hitsound dasar yang diputar jika sebuah note dengan sampleset Soft dan addition "hitclap" sukses diklik. Mohon diperhatikan bahwa hitsound ini hanya akan diputar pada *selected sampleset*. Jika beatmap-mu juga menggunakan sampleset yang lain, sampleset tersebut butuh berkas hitsound tambahan (meskipun kamu ingin menggunakan sampel suara yang sama), seperti dengan menambahkan berkas `normal-hitclap.wav` ketika menggunakan sampleset Normal.
```
Comment (daud-ahn) [inline wiki/Guides/Using_custom_hitsounds/id.md:42]:
```suggestion
Pada gambar yang ditunjukkan di atas, berkas pertama bernama `soft-hitclap.wav` dan akan menggantikan hitsound bawaan yang diputar jika sebuah not dengan sampleset Soft dan addition "hitclap" sukses diklik. Mohon diperhatikan bahwa hitsound ini hanya akan diputar pada *selected sampleset*. Jika beatmap-mu juga menggunakan sampleset yang lain, sampleset tersebut butuh berkas hitsound tambahan (meskipun kamu ingin menggunakan sampel suara yang sama), seperti dengan menambahkan berkas `normal-hitclap.wav` ketika menggunakan sampleset Normal.
```
`not` (`note`) here can be replaced with `hit object` i think (change the en version too)

File: wiki/Guides/Using_custom_hitsounds/id.md [hunk 3]
```diff
@@ -1,48 +1,68 @@
...
 
-Jangan lupa buka timing setup dan ubah hitsound ke *Custom*, dan pilih nomor yang lebih spesifik.
+## Menggunakan hitsound khusus
 
-![Tutorial Image 2](img/adding-custom-hitsounds.png "Tutorial Image 2")
+![](img/adding-custom-hitsounds.png "Memberi perintah pada osu! bagaimana cara menggunakan hitsound addition khusus")
+
+Supaya osu! dapat memutar hitsound khususmu secara benar, pastikan kamu mencentang opsi kedua, "Custom 1", seperti gambar di atas. Nama sampleset khusus dasar disingkat namanya menjadi `<SS>:C1` di mana `<SS>` merupakan huruf pertama pada grup sampleset seperti N (Normal), S (Soft), atau D (Drum).
+
+Perlu diingat bahwa kamu tidak perlu menambahkan hitsound khusus pada setiap suara pada sampleset. Dapat dilihat pada gambar pertama bahwa `soft-slidertick.wav` tidak ditambahkan, sehingga osu! akan memutar suara slider tick dengan hitsound dasar saat sampleset Soft digunakan pada beatmap tersebut.
```
Comment (daud-ahn) [inline wiki/Guides/Using_custom_hitsounds/id.md:50]:
```suggestion
Perlu diingat bahwa kamu tidak perlu menambahkan hitsound khusus pada setiap suara pada sampleset. Dapat dilihat pada gambar pertama bahwa `soft-slidertick.wav` tidak ditambahkan, sehingga osu! akan memutar suara slider tick dengan hitsound bawaan saat sampleset Soft digunakan pada beatmap tersebut.
```

File: wiki/Guides/Using_custom_hitsounds/id.md [hunk 4]
```diff
@@ -1,48 +1,68 @@
...
+
+## Menggunakan beberapa set hitsound khusus
+
+Terkadang sebuah lagu mungkin memiliki beberapa bagian yang di mana gaya musiknya berbeda dan sebuah grup hitsound tidak dapat cocok pada bagian tersebut. Pada kasus ini, sangat disarankan untuk menggunakan hitsound atau grup hitsound yang berbeda. Hal ini dapat dicapai dengan menambahkan angka pada ujung nama berkas hitsound sebagai berikut:
+
+`<sampleset>-<sound><#>.wav`
+
+dimana `<#>` berupa angka apapun. Editor osu! secara dasar mendukung angka dari 2 sampai 100, tetapi angka yang lebih besar dapat dibuat dengan mengedit berkas `.osu` jika dibutuhkan. Perlu diingat bahwa grup hitsound pertama tidak perlu dilabeli dengan angka "1" meskipun beberapa grup hitsound digunakan. Sehingga berkas `soft-hitclap1.wav` tidak akan bekerja dan berkas `soft-hitclap.wav` akan digunakan.
```
Comment (daud-ahn) [inline wiki/Guides/Using_custom_hitsounds/id.md:58]:
```suggestion
dimana `<#>` berupa angka apapun. Editor osu! secara bawaan mendukung angka dari 2 sampai 100, tetapi angka yang lebih besar dapat dibuat dengan mengedit berkas `.osu` jika dibutuhkan. Perlu diingat bahwa grup hitsound pertama tidak perlu dilabeli dengan angka "1" meskipun beberapa grup hitsound digunakan. Sehingga berkas `soft-hitclap1.wav` tidak akan bekerja dan berkas `soft-hitclap.wav` akan digunakan.
```

File: wiki/Guides/Using_custom_hitsounds/id.md [hunk 5]
```diff
@@ -1,48 +1,68 @@
...
+Pada gambar yang ditunjukkan di atas, berkas pertama bernama `soft-hitclap.wav` dan akan menggantikan hitsound dasar yang diputar jika sebuah note dengan sampleset Soft dan addition "hitclap" sukses diklik. Mohon diperhatikan bahwa hitsound ini hanya akan diputar pada *selected sampleset*. Jika beatmap-mu juga menggunakan sampleset yang lain, sampleset tersebut butuh berkas hitsound tambahan (meskipun kamu ingin menggunakan sampel suara yang sama), seperti dengan menambahkan berkas `normal-hitclap.wav` ketika menggunakan sampleset Normal.
 
-Jangan lupa buka timing setup dan ubah hitsound ke *Custom*, dan pilih nomor yang lebih spesifik.
+## Menggunakan hitsound khusus
 
-![Tutorial Image 2](img/adding-custom-hitsounds.png "Tutorial Image 2")
+![](img/adding-custom-hitsounds.png "Memberi perintah pada osu! bagaimana cara menggunakan hitsound addition khusus")
+
+Supaya osu! dapat memutar hitsound khususmu secara benar, pastikan kamu mencentang opsi kedua, "Custom 1", seperti gambar di atas. Nama sampleset khusus dasar disingkat namanya menjadi `<SS>:C1` di mana `<SS>` merupakan huruf pertama pada grup sampleset seperti N (Normal), S (Soft), atau D (Drum).
```
Comment (daud-ahn) [inline wiki/Guides/Using_custom_hitsounds/id.md:48]:
```suggestion
Supaya osu! dapat memutar hitsound khususmu secara benar, pastikan kamu mencentang opsi kedua, "Custom 1", seperti gambar di atas. Nama sampleset khusus bawaan disingkat namanya menjadi `<SS>:C1` di mana `<SS>` merupakan huruf pertama pada grup sampleset seperti N (Normal), S (Soft), atau D (Drum).
```

File: wiki/Guides/Using_custom_hitsounds/id.md [hunk 6]
```diff
@@ -1,48 +1,68 @@
...
+
+Untuk memastikan hitsound yang memiliki angka atau grup hitsound diputar secara benar, kamu perlu menambahkan sebuah *inherited timing point (garis hijau)* dan mengubah sampleset dari "Custom 1" menjadi opsi yang ada di bawahnya seperti gambar di bawah ini. Di sini kamu dapat memasukkan angka pada grup hitsound yang kamu ingin gunakan.
+
+![](img/using-multiple-hitsound-sets.png "Mengubah hitsound ke grup hitsound khusus kedua")
+
+Ketika sampleset dari *inherited timing point* diset ke `S:C2` seperti pada gambar di atas, seluruh hitsound dasar dan hitsound addition akan digantikan oleh hitsound khusus yang telah dinamakan seperti `soft-hitclap2.wav` jika ada. Hitsound tersebut akan terus dapat digunakan hingga *inherited timing point* yang menggunakan sampleset berbeda dicapai—pada gambar ini *inherited timing point* tersebut berada di `02:00:723`, di mana sampleset berubah kembali menjadi `S:C1`.
```
Comment (daud-ahn) [inline wiki/Guides/Using_custom_hitsounds/id.md:64]:
```suggestion
Ketika sampleset dari *inherited timing point* diset ke `S:C2` seperti pada gambar di atas, seluruh hitsound bawaan dan hitsound addition akan digantikan oleh hitsound khusus yang telah dinamakan seperti `soft-hitclap2.wav` jika ada. Hitsound tersebut akan terus dapat digunakan hingga *inherited timing point* yang menggunakan sampleset berbeda dicapai—pada gambar ini *inherited timing point* tersebut berada di `02:00:723`, di mana sampleset berubah kembali menjadi `S:C1`.
```

**Review by daud-ahn — COMMENTED** [review]
missed

**Review by Walavouchey — APPROVED** [review]

**Pull request merged**